link.go

```diff
@@ -28,6 +28,14 @@ type LinkAttrs struct {
 	Namespace    interface{} // nil | NsPid | NsFd
 }
 
+// NewLinkAttrs returns LinkAttrs structure filled with default values
+func NewLinkAttrs() LinkAttrs {
+	return LinkAttrs{
+		TxQLen: -1,
+		MTU:    -1,
+	}
+}
+
```

Comment:
I was pretty sure, that we can :/ Actually with `TxQLen` all works as expected, apart from setting it to `0`.